setup.cfg

```diff
@@ -45,7 +45,7 @@ combine_as_imports = true
 include_trailing_comma = true
 line_length = 88
 known_first_party = superset
-known_third_party =alembic,apispec,backoff,bleach,cachelib,celery,click,colorama,contextlib2,croniter,cryptography,dataclasses,dateutil,flask,flask_appbuilder,flask_babel,flask_caching,flask_compress,flask_login,flask_migrate,flask_sqlalchemy,flask_talisman,flask_testing,flask_wtf,geohash,geopy,humanize,isodate,jinja2,markdown,markupsafe,marshmallow,msgpack,numpy,pandas,parsedatetime,pathlib2,polyline,prison,pyarrow,pyhive,pytz,retry,selenium,setuptools,simplejson,sphinx_rtd_theme,sqlalchemy,sqlalchemy_utils,sqlparse,werkzeug,wtforms,wtforms_json,yaml
+known_third_party =alembic,apispec,backoff,bleach,cachelib,celery,click,colorama,contextlib2,croniter,cryptography,dateutil,flask,flask_appbuilder,flask_babel,flask_caching,flask_compress,flask_login,flask_migrate,flask_sqlalchemy,flask_talisman,flask_testing,flask_wtf,geohash,geopy,humanize,isodate,jinja2,markdown,markupsafe,marshmallow,msgpack,numpy,pandas,parsedatetime,pathlib2,polyline,prison,pyarrow,pyhive,pytz,retry,selenium,setuptools,simplejson,sphinx_rtd_theme,sqlalchemy,sqlalchemy_utils,sqlparse,werkzeug,wtforms,wtforms_json,yaml
```

Comment:
Why is it that every one of my PR flip flops this things?
![ferris](https://user-images.githubusercontent.com/487433/84720821-f6618300-af33-11ea-8898-b60e84441aab.gif)

Reply:
I'm actually about 0.3% as mad as Cameron is here. ^ :)

Reply:
This is probably because pre-commit is running on python 3.7 sometimes. `dataclasses` is only included in 3.7 and above, so it's counted as a third party lib in 3.6 and below. Until we deprecate 3.6 (likely in our 1.0 release because it's pretty breaking), this should stay as a known third party library

Reply:
ok, downgrading local to 3.6, `brew` makes it hard to have both 3.6 and 3.7 :(

Reply:
Oh wait, but that failed in GitHub actions CI, would GitHub actions be flip-flopping between 3.6 and 3.7?

Reply:
github actions is always running in 3.6 for doing linting i think. but if you regen the file on 3.7 and then it's linted in 3.6, errors will be thrown

Reply:
your CI is failing for a frontend issue now

Reply:
I installed `pyenv` and both `3.7.7` and `3.6.9` and both seem to agree (isort wants dataclasses at the top, it doesn't flip-flop between the 2).

I'm 100% sure I've seen Github action raise on `isort` earlier for this `dataclass` issue. Now I'm guessing that I got some older 3.6.x (?) on the GitHub infra maybe (?).

`pre-comit` only executes against files that you touch, and that went through ok locally, it's GitHub that was upset and got me to `isort` manually...

Anyways, tested on both sides, merging!

Reply:
Hmm, something still seems a bit screwy. when i run pre-commit on 3.6 i'm still getting it to flip around. Maybe @john-bodley our resident pylint/pre-commit guru knows what went wrong here?